README.md

```diff
@@ -1340,6 +1378,16 @@ The IOS operating system does not support:
 * enable
 * retransmit_count
 
+#### ios_interface
+
+##### 2960
+
+##### 4507r
+
+##### 6503
+
+The above devices do support the setting of `snmp trap mac-notification change added/removed`.
```

Comment:
Something like `The above devices do not support `mac_notification_added` and `mac_notification_removed` as the user may not know that these params relate to the command.

Reply:
changed